src/java/test/org/apache/zookeeper/test/ConnectStringParserTest.java

```diff
@@ -39,27 +44,44 @@ public void testMultipleServersChrootPath(){
         String servers = "10.10.10.1,10.10.10.2";
         assertChrootPath(chrootPath,
                 new ConnectStringParser(servers+chrootPath));
+        
+        servers = "[2001:db8:1::242:ac11:2]:2181,[2001:db8:85a3:8d3:1319:8a2e:370:7348]:5678";
```

Comment:
Would be useful to define static strings for the two IPv6 address, since you use them so often. Less chance to mistype when changing the test.

Reply:
good suggestion